src/rules/noUnusedVariableRule.ts

```diff
@@ -173,7 +238,7 @@ class NoUnusedVariablesWalker extends Lint.RuleWalker {
     public visitPropertyDeclaration(node: ts.PropertyDeclaration) {
         if (node.name != null && node.name.kind === ts.SyntaxKind.Identifier) {
             const modifiers = node.modifiers;
-            const variableName = (<ts.Identifier> node.name).text;
+            const variableName = (<ts.Identifier>node.name).text;
```

Comment:
Apologies, these have crept in because I automatically formatted the file using `tsfmt`